### PR TITLE
WIP Device List + OS

### PR DESCRIPTION
**Pull request by f0o**
- Adds OS Pictures
- Adds OS Abstraction Class & Loaders
- Mimics current devices list view

### CODE REVIEW

File: html/images/os/arch.svg [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (murrant) [inline html/images/os/arch.svg]: Can you remove this file, it isn't used ;)

Comment (f0o) [inline html/images/os/arch.svg]: haha ofc, I just blindly copied the old dir

**Comment by laf** [comment]
This good for a merge or are you waiting for more feedback @f0o ?

**Comment by murrant** [comment]
@f0o I have thought about this a little and I'm not so sure why DeviceOS should be its own class.  What kind of methods will it have?  I could see this from a poller/discovery angle, but should we plan for that at this time?

**Comment by laf** [comment]
I doubt a poller/disco rewrite will be using this code base. The only talks so far about it would be that it's python based rather than php.

**Comment by murrant** [comment]
Closing this, create a new PR if you have new code.

**Pull request closed without merging**